static/src/javascripts/boot-webpack.js

```diff
@@ -11,6 +11,14 @@ import capturePerfTimings from 'lib/capture-perf-timings';
 // eslint-disable-next-line camelcase,no-undef
 __webpack_public_path__ = `${config.page.assetsPath}javascripts/`;
 
+// Selectively attempt to pollute the global namespace with babel polfills.
+// The polyfills babel provides from core-js will be browser natives where
+// available anyway, so this shouldn't muck about with UAs that don't need it.
+if (!window.Promise) {
+    // used by webpack `import`
+    window.Promise = Promise;
```

Comment:
I don't see the polyfill as a local variable, does it mean this is redundant? i.e. `Promise` is _already_ being overwritten?